cfgov/unprocessed/apps/paying-for-college/js/dispatchers/get-model-values.js

```diff
@@ -36,37 +40,71 @@ function getExpensesValue( name ) {
 }
 
 /**
- * getConstantsValue - get the value of 'name' from the constantsModel
+ * getFinancialValue - get the value of 'name' from the financialModel
  * @param {string} name - Name of the property to retrieve
- * @returns {(number|string)} value of the property
+ * @returns {number|booleab} value of the property, or false if undefined
  */
-function getConstantsValue( name ) {
-  return constantsModel.values[name];
+function getFinancialValue( name ) {
+  if ( financialModel.values.hasOwnProperty( name ) ) {
+    return financialModel.values[name];
+  }
+  return false;
+
 }
 
 /**
- * getStateValue - gets the property from the application state model
- * @param {string} prop - The property name
- * @returns {string} The value of the property in the stateModel Object
+ * getProgramList - retrieve an alphabetical list of programs
+ * from the schoolModel
+ * @param {string} level - program level - 'undergrad' or 'graduate'
+ * @returns {array} an arry of objects containing program data
  */
-function getStateValue( prop ) {
-  return stateModel.values[prop];
+function getProgramList( level ) {
+  return schoolModel.getAlphbeticalProgramList( level );
 }
 
 /**
- * getAllStateValues - retrieves the entire values property (object) of the stateModel
- * @returns {Object} The values stored in the stateModel Object
+ * getProgramInfo - retrieve info based on pid
+ * @param {string} pid - Program ID
+ * @returns {object} Values of the program
  */
-function getAllStateValues() {
-  return { ...stateModel.values };
+function getProgramInfo( pid ) {
+  return schoolModel.getProgramInfo( pid );
+}
+
+/**
+ * getSchoolValue - get the value of 'name' from the schoolModel
+ * @param {string} name - Name of the property to retrieve
+ * @returns {(number|string|boolean)} value of the property, or false if undefined
+ */
+function getSchoolValue( name ) {
+  if ( schoolModel.values.hasOwnProperty( name ) ) {
+    return schoolModel.values[name];
+  }
+  return false;
+
 }
 
+/**
+ * getStateValue - gets the property from the application state model
+ * @param {string} prop - The property name
+ * @returns {stringboolean} The value of the property in the stateModel Object, or
+ *    false if undefined
+ */
+function getStateValue( prop ) {
+  if ( stateModel.values.hasOwnProperty( prop ) ) {
+    return stateModel.values[prop];
+  }
+  return false;
```

Comment:
I can live with empty string - I think `return false` is a habit I pick up from other languages which isn't as relevant here.

Alternatively, I might add a catch in the text output, since it would always be good to catch non-String values (like `null` and `false`) from ever hitting the DOM.